src/Raven.Server/Documents/Sharding/ShardedDocumentDatabase.cs

```diff
@@ -43,6 +49,9 @@ public ShardedDocumentDatabase(string name, RavenConfiguration configuration, Se
         ShardNumber = ShardHelper.GetShardNumberFromDatabaseName(name);
         ShardedDatabaseName = ShardHelper.ToDatabaseName(name);
         Smuggler = new ShardedDatabaseSmugglerFactory(this);
+
+        _logger = LoggingSource.Instance.GetLogger<ShardedDocumentDatabase>(Name);
+        _shardedDatabaseContext = serverStore.DatabasesLandlord.TryGetOrCreateDatabase(ShardedDatabaseName).DatabaseContext;
```

Comment:
this is wrong, you load the orchestrator inside the shard?
orchestrator doesn't have to reside on the same node as the shard

Reply:
right! sorry, done